### DISCUSSION TITLE
Cypress Exception Reporting

### DISCUSSION THREAD

**Discussion by alyoung1621** [cypress-io/cypress#28624] [Questions and Help]
I would like to disable Cypress exception reporting per the [docs.](https://docs.cypress.io/guides/references/advanced-installation#Opt-out-of-sending-exception-data-to-Cypress). If Cypress is already installed, will the variable take effect if I set `CYPRESS_CRASH_REPORTS`  to 0 before starting or do I have to uninstall Cypress, set the `CYPRESS_CRASH_REPORTS` variable to 0, and reinstall Cypress?

**Comment by portyu9** [comment]
Hi @alyoung1621 

You do **not** need to reinstall Cypress for this to take effect.

The current Cypress implementation checks `CYPRESS_CRASH_REPORTS` from `process.env` at runtime. So for exception reporting, it is sufficient for the environment variable to be present **before the Cypress process is launched**:

```bash
CYPRESS_CRASH_REPORTS=0 npx cypress run
```

or export it in the shell/environment that starts Cypress.

If Cypress is already running, restart that Cypress process after setting the variable so it inherits the updated environment.

The documentation's recommendation to set it before installation is a safe installation/setup instruction, but the actual reporting check does not require reinstalling an already downloaded Cypress binary.

Please mark this answer as accepted if it helped, thank you.
